srcjs/input_rate.js

```diff
@@ -260,6 +260,9 @@ var InputNoResendDecorator = function(target, initialValues) {
 
     this.lastSentValues = cacheValues;
   };
+  this.forget = function(name) {
+    delete this.lastSentValues[name];
```

Comment:
Split the name here

Reply:
@wch I'm trying to remember why we thought this was a problem? `name` isn't expected to be `name:type`, just the id.

Reply:
(Oh I see your other comment, I think I just need to try it)